examples/setixset.dx

```diff
@@ -0,0 +1,190 @@
+import sort
+
+'#### Misc instances
+
+instance {n} [Ix n] Eq n
+  (==) = \x y. ordinal x == ordinal y
+
+def lexicalOrder {n} [Ord n]
+  (compareElements:n->n->Bool)
```

Comment:
What kind of comparison do you expect here? Perhaps make the function return `Ordering` (as defined in the prelude)?

Reply:
I'm expecting the `(>)` or `(<)` operator as defined in the `Ord` typeclass.  This function returns a `Bool` because that's what the typeclass instance below needs.  We could always redefine `Ord` in terms of a single function that returns an `Ordering` - that might be less wasteful when calling e.g. `(>=)` and `(<=)`, which right now require calling both `(>)` and `(<)`.

Reply:
In any case, I updated the comment to explain this more clearly.